library/systemd/README.md

```diff
@@ -0,0 +1,76 @@
+# Systemd actions
+
+The following table describes the actions to perform over a systemd service and its associated socked (if any) when we try to start, stop, restart or reload the service.
+
+
+| Status | Start | Stop | Restart | Reload |
+|---|---|---|---|---|
+| Socket running | nothing | stop socket | stop socket and start service/socket (depending on start mode) | nothing |
+| Service running | nothing  | stop service  | stop service and start service/socket (depending on start mode) | reload service (if it support, otherwise restart) |
+| Socket & Service running | nothing | stop service and socket | stop service and start service/socket (depending on start mode) | reload service (if it support, otherwise restart) |
+| nothing runs | start service/socket (depending on start mode) | nothing | start service/socket (depending on start mode) | start service/socket (depending on start mode) |
+
+
+## Actions over a `SystemService`
+
+In systemd, it might happen that a service is compose by a set of units (services, sockets, paths and so on). This class `SystemService` is able to group those units and handle them together.
+
+### `SystemService` status
+
+* [1] A service is stopped when neither the systemd socket nor service are running
+* [2] A service is running when the systemd socket and/or service are running
+
+### Start
+
+* Target
```

Comment:
Please use some other word that does not have a meaning in systemd. "Goal" or "Intent"